clothing/dashboard_widgets.py

```diff
@@ -0,0 +1,30 @@
+from dashboard.components import TemplateWidgetComponent
+from ophasebase.models import Ophase
+from .models import Person, Order
+from django.utils.translation import ugettext_lazy as _
+
+
+class ClothingOrderWidget(TemplateWidgetComponent):
+    permissions = ['clothing.add_order']
+    name = _('Kleidung')
+    template_name = "clothing/dashboard/widget_orders.html"
+
+    def get_context_data(self):
+        context = super().get_context_data()
+
+        current_ophase = Ophase.current()
+        context['ophase_title'] = _('Ophase')
+        if current_ophase is not None:
+            context['ophase_title'] = str(current_ophase)
+
+            staff = Person.objects.filter(ophase=current_ophase)
```

Comment:
Please consider adding something like
```python
    @staticmethod
    def get_current(**kwargs):
        return Person.objects.filter(ophase=Ophase.current(), **kwargs)
```
to the Person model and use it where appropriate. Several other models provide such methods, too.